lib/Optimizer/Lower/Lower.cpp

```diff
@@ -1906,7 +1955,8 @@ bool glow::lowerNode(Function *F, Node *node, CompilationContext &cctx) {
     CASE_LOWER(LSTMUnit);
     CASE_LOWER(Broadcast);
     CASE_LOWER(LogSoftMax);
-    CASE_LOWER(HardSwish);
+    CASE_DEQUANT(Tanh, ElemKind::FloatTy);
+    CASE_LOWER_QUANT(HardSwish, ElemKind::FloatTy);
```

Comment:
One concern I have here is if we have fp16 conversion turned on. Lowering happens after fp16 conversion. So we'd end up with some ops such as Tanh/HardSwish that will be in fp32 even though fp16 conversion would have liked to convert them to fp16. Any thoughts here?